src/common/transformations/src/transformations/common_optimizations/ric_fusion.cpp

```diff
@@ -60,13 +66,23 @@ class Attribute {
                       });
     }
 
-    void set_callback(callback_t callback) {
-        m_callback = std::move(callback);
-    }
-
     // Apply callback to materialize RIC inside graph
-    void operator()(Input<Node> input) const {
-        m_callback(input, *this);
+    void materialize(Input<Node> input) const {
```

Comment:
it looks like we can also delete `callback_t`

Reply:
right, removed